src/parkapi_sources/converters/opendata_swiss/models.py

```diff
@@ -0,0 +1,200 @@
+
+    def to_osm_opening_day_format(self) -> str:
+        return {
+            self.MONDAY: 'Mo',
+            self.TUESDAY: 'Tu',
+            self.WEDNESDAY: 'We',
+            self.THURSDAY: 'Th',
+            self.FRIDAY: 'Fr',
+            self.SATURDAY: 'Sa',
+            self.SUNDAY: 'Su',
+        }.get(self, None)
+
+
+@validataclass
+class OpenDataSwissOperationTimeInput:
+    operatingFrom: Optional[str] = Noneable(StringValidator())
+    operatingTo: Optional[str] = Noneable(StringValidator())
+    daysOfWeek: Optional[list[str]] = Noneable(ListValidator(EnumValidator(OpenDataSwissOperationTimeDaysOfWeek)))
+
+
+@validataclass
+class OpenDataSwissPropertiesInput:
+    operator: str = StringValidator()
+    displayName: str = StringValidator()
+    address: Optional[OpenDataSwissAddressInput] = Noneable(DataclassValidator(OpenDataSwissAddressInput))
+    capacities: list[OpenDataSwissCapacitiesInput] = ListValidator(DataclassValidator(OpenDataSwissCapacitiesInput))
+    additionalInformationForCustomers: Optional[OpenDataSwissAdditionalInformationInput] = Noneable(
+        DataclassValidator(OpenDataSwissAdditionalInformationInput)
+    )
+    parkingFacilityCategory: str = StringValidator()
+    parkingFacilityType: Optional[str] = Noneable(StringValidator())
+    salesChannels: Optional[list[str]] = Noneable(ListValidator(StringValidator()))
+    operationTime: Optional[OpenDataSwissOperationTimeInput] = Noneable(DataclassValidator(OpenDataSwissOperationTimeInput))
+
+    def __post_init__(self):
+        for capacity in self.capacities:
+            if capacity.categoryType == OpenDataSwissCapacityCategoryTypeInput.STANDARD:
+                return
+        # If no capacity with type PARKING was found, we miss the capacity and therefore throw a validation error
+        raise ValidationError(reason='Missing parking spaces capacity')
+
+    def get_osm_opening_hours(self) -> str:
+        open_swiss_opening_times_by_weekday: dict[OpenDataSwissOperationTimeDaysOfWeek, list[str]] = defaultdict(list)
+        check_counter_24_7: int = 0
+        check_list_weekday: list[str] = []
+
+        # OSM 24/7 has no secs in its timeformat and no endtime 00:00, so we replace with 24:00 and remove the secs
+        closing_hh_mm_ss: list[str] = list(self.operationTime.operatingTo.replace('00:00:00', '24:00:00').split(':'))
+        opening_hh_mm_ss: list[str] = list(self.operationTime.operatingFrom.split(':'))
+        opening_time: str = f'{opening_hh_mm_ss[0]}:{opening_hh_mm_ss[1]}-{closing_hh_mm_ss[0]}:{closing_hh_mm_ss[1]}'
+
+        for days_of_week_input in self.operationTime.daysOfWeek:
+            # If it's open all day, add it to our 24/7 check counter, and we change opening_time to the OSM format
+            if opening_time == '00:00-24:00':
+                check_counter_24_7 += 1
+
+            # Add opening times to fallback dict
+            open_swiss_opening_times_by_weekday[days_of_week_input].append(opening_time)
```

Comment:
I'm not sure if that makes much sense. If I understand the format correctly, the times are always the same, so why listing them at all? In general, if I understand the format correctly, you can simplify this method a lot.

Reply:
The whole method for ```get_osm_opening_hours``` have now been simplified